Detections/SecurityEvent/UserAccountCreatedDeleted_10m.txt

```diff
@@ -0,0 +1,33 @@
+// Name: User Account Created and Deleted within 10mins
+// Description: User account created and then deleted within 10 minutes
+//
+// Id: 4b93c5af-d20b-4236-b696-a28b8c51407f
+//
+// Severity: Medium
+//
+// QueryFrequency: 10m
+//
+// QueryPeriod: 10m
+//
+// Data Source: #SecurityEvent
+//
+// Techniques: #Persistence, #Discovery, #LateralMovement, #Collection
+//
+// TimeFrame is the number of lookback mins, default is last 10mins
+let timeframe = 10m;
+SecurityEvent 
+| where TimeGenerated > ago(timeframe) 
+// A user account was created
+| where EventID == "4720"
+| where AccountType == "User"
+| project TimeGenerated, EventID, Activity, Computer, TargetUserName, UserPrincipalName, SubjectUserName, TargetSid, SubjectUserSid 
```

Comment:
Doing 
project creationTime=TimeGenerated
here, and project deletionTime=TimeGenerated below
would then make the later proposed filter
| where deletionTime - creationTime < 10m
easier to read/understand?

Reply:
yep, makes sense...was trying to stay within the Time naming schema.

Reply:
So, had to add this in otherwise we can get a negative timespan.

| where deletionTime - creationTime < 10m
| extend TimeDelta = deletionTime - creationTime
| where tolong(TimeDelta) >= 0